src/views/HomeView.vue

```diff
@@ -20,9 +28,12 @@ import axios from "axios";
 })
 export default class HomeView extends Vue {
   post: IPost | null = null;
-  async mounted() {
-    // await this.getPost();
-  }
+  maxLength = 5000;
+  textareaOptions: object = {};
+  textareaContent: textareaContent = {
+    input: "",
+    output: "",
```

Comment:
I'm not sure about the future implementation, but suppose you can leave it as 2 simple values:
`input` and `output` (without keeping them in `textareaContent` object.
Please change this if you find it helpful otherwise leave it without changes